tests/js/sentry-test/performance/utils.ts

```diff
@@ -12,26 +12,35 @@ export const EXAMPLE_TRANSACTION_TITLE = '/api/0/transaction-test-endpoint/';
 type AddSpanOpts = {
   endTimestamp: number;
   startTimestamp: number;
+  data?: Record<string, any>;
   description?: string;
   op?: string;
   problemSpan?: ProblemSpan;
   status?: string;
 };
 
+interface TransactionSettings {
+  duration?: number;
+}
 export class TransactionEventBuilder {
   TRACE_ID = '8cbbc19c0f54447ab702f00263262726';
   ROOT_SPAN_ID = '0000000000000000';
   #event: EventTransaction;
   #spans: RawSpanType[] = [];
 
-  constructor(id?: string, title?: string, problemType?: IssueType) {
+  constructor(
+    id?: string,
+    title?: string,
+    problemType?: IssueType,
+    transactionSettings?: TransactionSettings
+  ) {
     this.#event = {
       id: id ?? 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
       eventID: id ?? 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
       title: title ?? EXAMPLE_TRANSACTION_TITLE,
       type: EventOrGroupType.TRANSACTION,
       startTimestamp: 0,
-      endTimestamp: 0,
+      endTimestamp: transactionSettings?.duration ?? 0,
```

Comment:
A little surprising that it's `duration` and not `endTimestamp` but not a big deal

Reply:
I don't actually want to map 1:1, I want a simpler api when using for testing. If there was a startTimestamp then I'd have this as `transactionSettings?.startTimestamp + ... duration` in order to make the use of this builder simpler.